src/main/resources/default/kb/de/user/ftp-import.html.pasta

```diff
@@ -0,0 +1,85 @@
+<k:article code="EAI5V" lang="de" parent="SLEOK"
+           title="Verbindung zum FTP(S)-Server und automatischer Start von Jobs" permissions="flag-logged-in">
+    <i:block name="description">Erklärt wie Jobs via FTP(S) Verbindung gestartet werden können.</i:block>
+
+    <k:section heading="Einrichtung des Zugriffs zum FTP-Server">
+        <p>
+            Sie können Dateien per FTP(S)-Upload übergeben um diese etwa im Medienpool abzulegen oder automatisch
+            einen Verarbeitungsjob starten zu lassen.
+        </p>
+        <p>
+            Im Folgenden finden Sie die dafür notwendigen Informationen.
+        </p>
+    </k:section>
+    <k:section heading="Protokoll und Serveradresse">
+        <p>
+            Es wird der Einsatz von SFTP empfohlen (Secure File Transfer Protocol, SSH File Transfer Protocol) - Port
+            22:
```

Comment:
Port 22 ist der standardmäßige Port. Das kann aber auch ein anderer sein. Gleiches gilt für FTP .

Reply:
@fhaScireum Du meinst pro Produkt konfiguriert/eingerichtet? Das können wir aber nicht irgendwo auslesen und dynamisch in den KBA schreiben, oder?

Reply:
@mbrsci müssten das nicht die Settings `storage.layer3.roots.ftp.port` und `storage.layer3.roots.sftp.port` sein?